docs/about.md

```diff
@@ -7,20 +7,32 @@
       display: none !important;
     }
 </style>
-  
+
 # About
 
-GTFS.org is the central documentation platform for the General Transit Feed Specification. This site and its contents are maintained by [MobilityData](https://mobilitydata.org/) and the GTFS community. 
+GTFS.org is the central documentation platform for the General Transit Feed Specification. This site and its contents are maintained by [MobilityData](https://mobilitydata.org/) and the GTFS community.
+
+Thank you to [Andrew Byrd](https://www.linkedin.com/in/byrdandrew) for purchasing and lending use of the GTFS.org domain name and providing leadership for the community.
+
+<br>
+
+##How can I edit content on gtfs.org?**
```

Comment:
My intuition is that most people will want to just open an issue than go through a PR process.

I think the title of this section can be:
Title: How can I contribute to gtfs.org?"
First section: Submit an issue or a content addition idea 
Second section: Suggest a change with a Pull Request

Reply:
Thanks @isabelle-dr ! Great point. I've updated the doc - let me know what you think.